src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +20,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
Make sure to handle the case where some of the CompletableFutures could complete exceptionally. As it stands, if any of the futures complete exceptionally, then the exception details are lost. Consider adding exceptionally() callbacks to handle exceptions.